target/linux/ramips/image/mt7621.mk

```diff
@@ -768,6 +768,16 @@ define Device/cudy_wr2100
 endef
 TARGET_DEVICES += cudy_wr2100
 
+define Device/cudy_r700
+  $(Device/dsa-migration)
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := R700
+  IMAGE_SIZE := 15872k
+  UIMAGE_NAME := R11
```

Comment:
I think this is version R29.
```
  UIMAGE_NAME := R29
  SUPPORTED_DEVICES += R29
```
Your signed image contains: `supported_devices":["R29"]`